### PR TITLE
Fix a few typos in the comments

### PR DESCRIPTION
**Pull request by Evangelink**
Fix a few typos in the code comments.

### CODE REVIEW

File: src/Elmish.WPF/Binding.fs [hunk 1]
```diff
@@ -21,7 +21,7 @@ type internal OneWaySeqLazyData<'model, 'a, 'b, 'id when 'id : equality> =
     Equals: 'a -> 'a -> bool
     GetId: 'b -> 'id
     ItemEquals: 'b -> 'b -> bool }
-
```
Comment (Evangelink) [inline src/Elmish.WPF/Binding.fs:24]: Sorry, I will revert all of these whitespace changes (I forgot again to disable the VS extension removing them on save).

**Comment by TysonMN** [comment]
Thanks!

File: src/Elmish.WPF/Binding.fs (hunk 1 above)
Comment (TysonMN) [reply src/Elmish.WPF/Binding.fs]: I prefer to have no trailing spaces at the end of a line, so your PR removes some of them, then feel free to leave them in (as long as the diff is easy to understand).

Comment (Evangelink) [reply src/Elmish.WPF/Binding.fs]: I will do a separate PR with trailing space removal.

**Pull request merged**